### PR TITLE
Get value of fullscreen status in JavaScript

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
When adding some function / text eg. ```showAnswer();``` then user have to turn on full screen.
But if user turn off full screen then for hiding that buttons / text, there is way to know if full screen or not in JavaScript. 

## Fixes
Fixes _Link to the issues._

## Approach
Using ```JavascriptInterface```, passing ```Action Bar isShowing``` 

## How Has This Been Tested?
Tested on emulator 

```false``` when ```Action Bar``` showing

full

```true``` when ```Action Bar``` not showing

full2

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 1]
```diff
@@ -3497,5 +3497,11 @@ public int ankiGetCardLapses() {
         public long ankiGetCardDue() {
             return mCurrentCard.getDue();
          }
+
+        @JavascriptInterface
+        public boolean ankiIsInFullscreen() {
+            isInFullscreen = !getSupportActionBar().isShowing();
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:3503]: This is now used in 3 places - might be best to extract it to a method inside `AbstractFlashcardViewer`

Comment (infinyte7) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:3503]: I tried to extract inside ```ocreate``` but AnkiDroid crash everytime.

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java]: most of the interesting stuff in AnkiDroid activities needs to be done in (or called from / after) onCollectionLoaded

Comment (infinyte7) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java]: Understood, But now I have extracted it inside ```onCollectionLoaded``` but always getting ```false``` value.

So, may be extracting to function is good.

**Pull request merged**